src/content/docs/en/linux/ssh.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: SSH Secure Shell
+description: "An overview of the SSH utility."
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+Secure Shell ("SSH") is one of the most common utilities when access to a remote device is needed. It's encrypted, powerful, and simple to understand.
+
+## SSH overview
+SSH uses a client/server model where one computer acts as an SSH server[^1] and allows other computers to connect.

```

Comment:
SSH uses a client/server model. One computer acts as a server[^1] and allows other computers (clients) to connect.